ref/src/CMakeLists.txt

```diff
@@ -3,10 +3,16 @@ netCDFModule.f90
 print_affinity.c
 runnable.c
 yppm_core.F90
+interpolate.f90
 yppm_driver.f90
 )
 
+list (APPEND test_src_files
+interpolateTest.f90
+)
```

Comment:
Everything related to testing needs to go under `ref/test`.  This content should be in `ref/test/CMakeLists.txt`

Reply:
Done.